### ISSUE TITLE
Not opening browser

### ISSUE THREAD

**Issue by adamreisnz** [nobuhito/atom-export-html#4]
Hi, just found your package and was hoping it would open the browser straight away as advertised, but this doesn't work. The HTML is opened in Atom instead, so that's kind of pointless for me as I want to be able to print the file.

Are your relying on the OS's default file handler to "open in browser"? Atom is my default opener for HTML files, so that won't work.

**Comment by adamreisnz** [comment]
Yeah I can confirm this. When I change the HTML files to be opened by browser by default, it works.

Could you do a more robust check somehow, or allow us to specify the app to use for opening the files? That way we can force to use the browser even if it's not the default file handler for HTML files.

**Comment by nobuhito** [comment]
@adambuczynski  Please try v.2.0.2.

**Comment by adamreisnz** [comment]
@nobuhito tried with 2.0.2, but still opens the output HTML in Atom itself (which is my default HTML handler).

**Comment by nobuhito** [comment]
Sorry for the late reply. Is it wrong with the current version?

Does it occur even if I select Atom with "Settings> OpenIn"?
